site/content/en/docs/drivers/kvm2.md

```diff
@@ -34,7 +34,6 @@ The `minikube start` command supports 5 additional KVM specific flags:
 * `Machine didn't return an IP after 120 seconds` when firewall prevents VM network access [#3566](https://github.com/kubernetes/minikube/issues/3566)
 * `unable to set user and group to '65534:992` when `dynamic ownership = 1` in `qemu.conf` [#4467](https://github.com/kubernetes/minikube/issues/4467)
 * KVM VM's cannot be used simultaneously with VirtualBox  [#4913](https://github.com/kubernetes/minikube/issues/4913)
-* On some distributions, libvirt bridge networking may fail until the host reboots
```

Comment:
I'm working with libvirt since 2013 and I never seen such issue. Maybe the issue is creating the network without presisting it. This should not happen if you follow the libvirt network configuration docs.

Minikube had a bug before 1.34 when it was deleting the libvirt network. This could result in missing network after you delete a cluster.

Deleting this looks good since we don't have any evidence and it may not be relevant for many years.

Reply:
This was added in https://github.com/kubernetes/minikube/pull/4989 - there was no discussion on this item.